reference/resources/felixconfig.md

```diff
@@ -124,6 +125,25 @@ spec:
 | min      | Minimum index to use | 1-250             | int    |
 | max      | Maximum index to use | 1-250             | int    |
 
+#### AWS IAM role/policy for source-destination-check configuration
+
+Setting `awsSrcDstCheck` to `Disable` will automatically disable source-destination-check on EC2 instances in a cluster, provided necessary IAM roles and profiles are set. Check that the profile assigned to IAM role of cluster nodes contain the statement similar to the following:
```

Comment:
Rather than "check that" I might re-orient this to be more of a "must" statement?